src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const moneyTotal = days * 40;
+
+  if (days >= 7) {
+    return moneyTotal - 50;
+  } else if (days >= 3) {
+    return moneyTotal - 20;
+  }
```

Comment:
40, 50, and 20 are magic numbers
try to keep all numbers like these in variables for more readable code